lab5.py

```diff
@@ -0,0 +1,62 @@
+
+class Sneakers:
+    def __init__(self, brand, size, color, price, quantity, material, numberOfSales):
```

Comment:
Change argument name numberOfSales to numberofsales or number_of_sales to comply with PEP8